cap-ci/cap-pre-release.yaml

```diff
@@ -49,3 +49,8 @@ s3:
   bucket: kubecf
   region: us-west-2
   regexp: kubecf-bundle-v(.*).tgz
+schedule:
```

Comment:
We should have a boolean flag to control whether scheduling is on/off. `enabled: true`

Also, a comment here like ("if scheduling is enabled, nightlies will be run and triggering will be disabled on ...")

Reply:
DONE.